### PR TITLE
new package: pciutils

### PR DESCRIPTION
**Pull request by a-schaefers**
## Description of package
This package build I nabbed a while back from @konimex 's overlay. Not sure as to whether it is okay as-is or needs any work? I really like it much better than the default lspci command and am sure it will help a lot of newer users who are trying to build kernels.
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/pciutils/depends [hunk 1]
```diff
@@ -0,0 +1 @@
+linux-headers make
```
Comment (dylanaraps) [inline community/pciutils/depends]: This file can be removed.

File: community/pciutils/build [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+mk() {
+    make \
+        PREFIX=/usr/local \
+        ZLIB=no \
+        CC="${CC:-gcc}" \
+        OPT="$CFLAGS" \
+        SHAREDIR=/usr/share/hwdata \
+        MANDIR=/usr/share/man \
+        SBINDIR=/usr/local/bin \
```
Comment (dylanaraps) [inline community/pciutils/build]: `/usr` and `/usr/bin`?

File: community/pciutils/build [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+mk() {
+    make \
+        PREFIX=/usr \
+        ZLIB=no \
```
Comment (dylanaraps) [inline community/pciutils/build]: Any reason for this?

Comment (a-schaefers) [reply community/pciutils/build]: i dont know, as i said I nabbed the package from elsewhere, wanted to post it here and get it up to par for the main repo

Comment (dylanaraps) [reply community/pciutils/build]: Does it build fine without it? Sounds like it'll link to system zlib if omitted.

Comment (a-schaefers) [reply community/pciutils/build]: check the latest and comment. thanks

**Comment by a-schaefers** [comment]
I have no idea how to package this one. Closing.

**Pull request closed without merging**